File: PessoaPython/gato.py
# CLASSE Gato
# Trabalho de Gustavo Medeiros Brasil para a matéria Paradigmas de Linguagens de Programação
# Código disponível em https://github.com/gusBrasil/paradgimas-ling-prog

class Gato:

# Atributos da Classe Gato
    def __init__(self, nome, sexo, idade, peso):
        self.nome = nome
        self.sexo = sexo
        self.idade = idade
        self.peso = peso

    # Métodos da Classe Gato
    def Feliz(self):
        print('Estado: feliz!')

    def Triste(self):
        print('Estado: triste!')

    def Doente(self):
        print('Estado: doente!')
    
    def Faminto(self):
        print('Estado: faminto!')

    def Raiva(self):
        print('Estado: raiva!')

    def ExibirGato(self):
        print('Gato:', self.nome, self.sexo, self.idade, self.peso)

File: PessoaJava/Gato.java
// Trabalho de Gustavo Medeiros Brasil para a matéria Paradigmas de Linguagens de Programação
// Código disponível em https://github.com/gusBrasil/paradgimas-ling-prog

// Classe Gato
//package classe;

public class Gato {
    // Atributos da Classe Gato
    public String nome;
    public String sexo;
    public String idade;
    public String peso;

    // Métodos da Classe Gato
    public void feliz() {
        System.out.println("Estado: feliz!\n");
    }

    public void triste() {
        System.out.print("Estado: triste!\n");
    }

    public void doente() {
        System.out.print("Estado: doente!\n");
    }

    public void faminto() {
        System.out.println("Estado: faminto!\n");
    }

    public void raiva() {
        System.out.println("Estado: raiva!\n");
    }

    public void exibirGato() {
        System.out.println("Gato: " + nome + " " + sexo + " " + idade + " " + peso);
    }
}
